src/components/pages/StudyCase/EditStudyCasePage.jsx

```diff
@@ -41,36 +40,49 @@ class EditStudyCasePage extends React.Component {
   }
 
   componentWillMount() {
-    dispatch(getStudyCases({ id: this.props.studyCases.detailId }));
-    dispatch(getCategories({ type: 'solution' }));
+    dispatch(getStudyCases({
+      id: this.props.studyCases.detailId,
+      include: [
+        'category',
+        'cities',
+        'comments',
+        'country',
+        'documents',
+        'external-sources',
+        'impacts',
+        'impacts.external-sources',
+        'impacts.category',
+        'photos',
+        'project-bmes',
+      ],
+    }));
+    dispatch(getCategories({ type: 'Solution' }));
     dispatch(getBmes({
-      pageSize: 9999,
-      pageNumber: 1
+      pageSize: 999,
+      pageNumber: 1,
```

Comment:
It was just carryover from the current style (it was also not obvious to me why this decision happened). Removing.